darshan-util/pydarshan/darshan/__init__.py

```diff
@@ -3,7 +3,7 @@
 PyDarshan also provides a suite of analysis utilities.
 """
 
-__version__ = '3.3.1.0'
+__version__ = '3.3.1.1'
```

Comment:
I think, because you are changing the source proper in i.e., `discover_darshan.py`, it may indeed be the right choice to release a newer version. I guess you and Shane have a convention for what the different numbers/positions mean. If we weren't changing the source at all, and just rebuilding wheels, then only incrementing the build number on the wheels with `-1` might suffice, but probably not in this case.

One thing you might try, to save time, is upload a few test wheels to https://test.pypi.org/ first and ask the HPC center folks to point their `pip` installs at that location (`-i URL` flag I think?) before doing the actual release.